inst/extdata/french_broad_data.csv

```diff
@@ -0,0 +1,291 @@
+"date.time","DO.obs","DO.deficit","depth","k.O2","light"
+NA,"mg L^-1","mg L^-1","m","m d^-1","J m^-2 s^-1"
+"date.time","DO.obs","DO.deficit","depth","k.O2","light"
+"2012-08-30 00:00:00",7.39,2.67581239853204,1.5,10,0
```

Comment:
I owuld do everything in ST and not DT.  The probles (at least the 2 I looked at) are UTC, so it is easy to conert to standard time and you do not need to worry about the light function.
On Apr 24, 2015, at 3:11 PM, Alison Appling <notifications@github.com<mailto:notifications@github.com>> wrote:


In inst/extdata/french_broad_data.csv<https://github.com/USGS-R/streamMetabolizer/pull/25#discussion_r29086269>:

> @@ -0,0 +1,291 @@
> +"date.time","DO.obs","DO.deficit","depth","k.O2","light"
> +NA,"mg L^-1","mg L^-1","m","m d^-1","J m^-2 s^-1"
> +"date.time","DO.obs","DO.deficit","depth","k.O2","light"
> +"2012-08-30 00:00:00",7.39,2.67581239853204,1.5,10,0


good idea. it's just a bunch of strings, so i could tack on an " EST" to every date string. less sure how/whether do handle the "EDT" period - do you know what needs to be done to get that right?

—
Reply to this email directly or view it on GitHub<https://github.com/USGS-R/streamMetabolizer/pull/25/files#r29086269>.